README.md

```diff
@@ -27,3 +27,7 @@ pip install -r requirements.txt
 cd src
 python2.7 -m main.py
 ```
+
+## How to use Engine Application
+1. For moving any vertex of Graph just "capture it" with left button mouse
```

Comment:
We do not use terms "vertex" or "edge" within the progect. Because its a game we use "point" and "line". Also we call graph a map. I would suggest the next line:
1. For moving of any map point just grab it with left mouse button than move and drop where you want.